setup.py

```diff
@@ -9,7 +9,11 @@
     long_description = readme_md.read()
 
 extras_require = {
-    'tensorflow': ['tensorflow~=1.15', 'tensorflow-probability~=0.8', 'numpy~=1.16'],
+    'tensorflow': [
+        'tensorflow~=2.0',
+        'tensorflow-probability~=0.8',
+        'numpy<2.0,>=1.16.0',
```

Comment:
do we need to keep this around given: https://github.com/tensorflow/tensorflow/blob/5364121e858be715439c328c5f67de8441050f4b/tensorflow/tools/pip_package/setup.py#L61

Reply:
Yeah, we can simplify that to just `numpy~=1.16`. Good point!